Core/SFContentBlockerManagerExtension.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  SFContentBlockerManagerExtension.swift
+//  DuckDuckGo
+//
+//  Copyright © 2017 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+
+import SafariServices
+
+extension SFContentBlockerManager {
+    
+    struct Constants {
+        static let identifier = "com.duckduckgo.DuckDuckGo.ContentBlockerExtension"
+    }
+    
+    public static func reloadContentBlocker() {
+        reloadContentBlocker(withIdentifier: Constants.identifier) { error in
+            if let error = error {
+                Logger.log(text: "Could not reload content blocker in Safari due to \(error)")
+                return
+            }
+            Logger.log(text: "Content blocker rules for Safari reloaded")
+        }
```

Comment:
I find the content blocking extension a real pain to debug. But if I open the safari dev console and connect it to the simulator's safari session I can see that the content blocker is working as expected.
<img width="984" alt="safari_console" src="https://user-images.githubusercontent.com/1652407/29043734-505971da-7bb4-11e7-803b-0184e1c8859a.png">